core-bundle/tests/EventListener/DataContainer/RecordPreviewListenerTest.php

```diff
@@ -32,19 +32,21 @@ public static function tearDownAfterClass(): void
     /**
      * @dataProvider loadDataContainer
      */
-    public function testRegistersDeleteCallbackOnDeletableDataContainers(string $table, bool $notDeletable, ?array $expected): void
+    public function testRegistersDeleteCallbackOnDeletableDataContainers(string $table, string $dataContainer, bool $notDeletable, ?array $expected): void
     {
         $GLOBALS['TL_DCA'][$table]['config']['notDeletable'] = $notDeletable;
+        $GLOBALS['TL_DCA'][$table]['config']['dataContainer'] = $dataContainer;
 
         $framework = $this->mockContaoFramework();
         $connection = $this->createMock(Connection::class);
 
         $listener = new RecordPreviewListener($framework, $connection);
         $listener->registerDeleteCallbacks($table);
 
-        if ($notDeletable) {
+        if ($notDeletable || 'Table' !== $dataContainer) {
```

Comment:
This line still says `'Table' !== $dataContainer`. Intentionally?

Reply:
I was still working on the tests, please see the d3a988a